doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -137,7 +137,7 @@ The required number of connection points to a VM is described in `e.cap.004` abo
 
 <p align="center"><b>Table 2-3:</b> Reference Model Requirements: Cloud Infrastructure Software Profile Requirements</p>
 
-**(1)** [Workload Transition Guidelines](../chapters/appendix-a.md) might have other interfaces (such as SR-IOV VFs to be directly passed to a VM) or NIC-specific drivers on guest machines transiently allowed until more mature solutions are available with an acceptable level of efficiency to support telecom workloads (for example regarding CPU and energy consumption).
+**(1)** Workloads might have other interfaces (such as SR-IOV VFs to be directly passed to a VM) or NIC-specific drivers on guest machines transiently allowed until more mature solutions are available with an acceptable level of efficiency to support telecom workloads (for example regarding CPU and energy consumption).
```

Comment:
@karinesevilla in PR#2445 this is deleted -- with the "exception" gone "transiently allowed" doesn't apply.

Reply:
@goyal Yes agree, last part of the sentence removed